src/main/java/seedu/financialplanner/cashflow/Budget.java

```diff
@@ -32,50 +51,104 @@ public static void updateBudget(double budget) {
         assert initialBudget == budget : "Initial budget should be equal to updated budget";
     }
 
+    /**
+     * Returns the current budget.
+     *
+     * @return The current budget.
+     */
```

Comment:
I dont think javadoc  for getters and setters are necessary  :(((((((((((((((((((((((